### ISSUE TITLE
Add RelatedField

### ISSUE THREAD

**Issue by lamenezes** [lamenezes/simple-model#3]
Sometimes it's necessary to link on model to another such as:

```python
class Product(SimpleModel):
    fields = ('name', 'brand',  'attributes')
    attributes = fields.RelatedField(ProductAttribute)


class ProductAttribute(SimpleModel):
    fields = ('name', 'description', 'value')
```

It makes easier to create composed models as above by just doing:

```python
>>> product_data = {
...    'name': 'foo prod',
...    'brand': 'bar brand',
...    'attributes': [{
...        'name': 'baz attr',
...        'description': "i'm a baz attribute",
...        'value': 69
...    },
...    {
...        'name': 'qux attr',
...        'description': "i'm a qux attribute",
...        'value': 6.9
...    }]
...}

>>> product = Product(**product_data)
>>> product.attributes
[ProductAttribute(name='baz attr', ...), ProductAttribute(name='qux attr', ...)]
```

**Comment by Marlysson** [comment]
@lamenezes Hello.. Can I contribute in this?

What dificult of this?
Which steps should be done to doing this task..

**Comment by lamenezes** [comment]
Hi, this issue was labeled as **hard** because it requires knowledge on python descriptors. Also, I can't offer much help on this specific issue without solving the real problem myself.

Perhaps it would be better if you picked issue #2. It is simpler and I believe I can guide you through it.

**Comment by Marlysson** [comment]
Ok, without problems. I will take a look in #2

**Comment by lamenezes** [comment]
Solved on 1.0 branch
